todo.sh

```diff
@@ -6,57 +6,69 @@
 # like so $lib_dir/someProjectScript.sh
 this_path=${BASH_SOURCE[0]}
 lib_dir="$(dirname "$this_path")"
-
 export todo_task_file="${TODO_TASK_FILE:-$lib_dir/.todo}"
```

Comment:
It doesn't matter but if you set these after you source the file the variable will be available when calling the function without `export`. Since running a script creates it's own instance of the environment this will be reset after the application exists but I tend to be careful when exporting environment variables to someones shell, that can mess up a lot of things :)